.github/workflows/monitoring-CI.yml

```diff
@@ -49,6 +49,36 @@ jobs:
         working-directory: "./monitoring"
         run: pnpm test
 
+      - name: Set up synthetics pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          package_json_file: "synthetics/package.json"
+
+      - name: Setup synthetics node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: "./synthetics/.nvmrc"
+          cache: pnpm
+          cache-dependency-path: |
+            synthetics/pnpm-lock.yaml
+            cdk/pnpm-lock.yaml
```

Comment:
Is Node happy with being setup twice? I'm guessing the cache might not be working correctly?